Source/Resource+UIImage.swift

```diff
@@ -0,0 +1,29 @@
+//
+//	Legal Notice! DB Systel GmbH proprietary License!
+//
+//	Copyright (C) 2017 DB Systel GmbH
+//	DB Systel GmbH; Jürgen-Ponto-Platz 1; D-60329 Frankfurt am Main; Germany; http://www.dbsystel.de/
+
+//	This code is protected by copyright law and is the exclusive property of
+//	DB Systel GmbH; Jürgen-Ponto-Platz 1; D-60329 Frankfurt am Main; Germany; http://www.dbsystel.de/
+
+//	Consent to use ("licence") shall be granted solely on the basis of a
+//	written licence agreement signed by the customer and DB Systel GmbH. Any
+//	other use, in particular copying, redistribution, publication or
+//	modification of this code without written permission of DB Systel GmbH is
+//	expressly prohibited.
+
+//	In the event of any permitted copying, redistribution or publication of
+//	this code, no changes in or deletion of author attribution, trademark
+//	legend or copyright notice shall be made.
+//
+
+import Foundation
+import UIKit
```

Comment:
Maybe check the @available keyword or check this [if os(Linux)](https://github.com/dbsystel/DBNetworkStack/blob/f7ae0c6440ffbead23998cb9cd1e8bcd5d23fb65/Source/URLSessionNetworkAccess.swift#L37) statement